### PR TITLE
Fixed loading of properties. Added sidebar of properties info from fullscreen

### PR DESCRIPTION
**Pull request by FuriousLlama**

### CODE REVIEW

File: source/frontend/src/components/common/mapFSM/useLocationFeatureLoader.tsx [hunk 1]
```diff
@@ -70,23 +70,19 @@ const useLocationFeatureLoader = () => {
           });
         }
 
-        // If not selecting and there is no other information availiable
-        if (!isSelecting) {
-          const municipalityFeature = await adminLegalBoundaryLayerServiceFindOneMunicipality(
```
Comment (devinleighsmith) [inline source/frontend/src/components/common/mapFSM/useLocationFeatureLoader.tsx:75]: Hmm, so now when we are querying for a feature we will make a request to the municipality layer as well? is that required?

Comment (FuriousLlama) [reply source/frontend/src/components/common/mapFSM/useLocationFeatureLoader.tsx:75]: We should figure out what information is required when processing a draft drop. Right now, the information is loaded twice. Once on click and another on the selected properties view

Comment (devinleighsmith) [reply source/frontend/src/components/common/mapFSM/useLocationFeatureLoader.tsx:75]: Please log some tech debt for that so we can keep track of these map related issues.

**Review by devinleighsmith — APPROVED** [review]

File: source/frontend/src/components/common/mapFSM/useLocationFeatureLoader.tsx (hunk 1 above)
Comment (FuriousLlama) [reply source/frontend/src/components/common/mapFSM/useLocationFeatureLoader.tsx:75]: I did. Once we remove IMapProperty we will have to addresss it

**Pull request merged**